truffle/src/main/ruby/core/rubinius/api/shims/mirror.rb

```diff
@@ -31,6 +30,7 @@ def self.module_mirror(obj)
         end
       end
     end
+    Truffle::Primitive.install_rubinius_primitive self, :module_mirror
```

Comment:
In this particular case, it's intended to not be obvious, since it would be likely wrong with a `module function` (defined on both instance and class level) as self could be either a module or the surrounding self.
If it is just a singleton/class method, then it might be right to want it and currently there is `constuctor` for this (which is literally `onSingleton + needsSelf`), which was appropriately named when I looked at current use-cases.